packages/odyssey-react-mui/src/labs/FileUploadPreview.tsx

```diff
@@ -128,8 +128,13 @@ const FileUploadPreview = ({
 }: FileUploadPreviewProps) => {
   const odysseyDesignTokens = useOdysseyDesignTokens();
 
+  if (!fileNames.length) {
+    return null;
+  }
```

Comment:
Can we just ternary the one `return`? I might miss this if I'm not looking for it.

And do we really wanna render nothing?

Reply:
Was torn on this honestly. I can move this logic to the parent component actually. Probably makes more sense there